frontend/src/hooks/api/actions/useApiTokensApi/useApiTokensApi.ts

```diff
@@ -12,8 +13,12 @@ const useApiTokensApi = () => {
         propagateErrors: true,
     });
 
-    const deleteToken = async (secret: string) => {
-        const path = `api/admin/api-tokens/${secret}`;
+    const deleteToken = async (secret: string, project?: string) => {
+        const path = formatApiPath(
+            `api/admin/api-tokens/${secret}${
+                Boolean(project) && project !== '' ? `/${project}` : ''
```

Comment:
Boolean(project) should be enough, because Boolean(empty string) should return false and Boolean(non empty string) returns always true